### ISSUE TITLE
Elixir Project Run: No file named 

### ISSUE THREAD

**Issue by Istiakmorsalin** [KronicDeth/intellij-elixir#618]
Whenever I am trying to run my Elixir project on Intellij idea in mac OS(Elixir installed by brew) i get

> /usr/local/Cellar/elixir/1.4.1/bin/elixir "" ""
> No file named 

My run configuration is:

![screen shot 2017-02-14 at 5 53 43 pm](https://cloud.githubusercontent.com/assets/2936695/22928687/f0d43818-f2e1-11e6-961f-eadc4db697b0.png)

project structure is:

![screen shot 2017-02-14 at 5 54 22 pm](https://cloud.githubusercontent.com/assets/2936695/22928701/fdddb610-f2e1-11e6-8839-b5c46d9047cd.png)

**Comment by KronicDeth** [comment]
@Istiakmorsalin did you setup the project using [Import Project from External Model](https://github.com/KronicDeth/intellij-elixir/blob/master/README.md#import-project-from-external-model)?  Some users that open an Elixir project without using Import Project from External Model have reported similar issues in the past.

This looks like you don't have a `mix` path set.  Here's what an "empty" Elixir Mix Run Configuration output looks like:

```
/usr/local/Cellar/elixir/1.3.2/bin/elixir /usr/local/bin/mix ""
```

So, the first `""` in your output is the `mix` path, but you don't have that set.  Looking through the code, I think the mix path can only be set on importing the project (I will count it as a bug that it's not accessible in any of the UIs, but we'll open that as a separate bug), so re-import the project using  [Import Project from External Model](https://github.com/KronicDeth/intellij-elixir/blob/master/README.md#import-project-from-external-model)

**Comment by oinopion** [comment]
I had the same problem, you can set mix path in "Other settings > Elixir External Tools".

**Comment by ciroque** [comment]
Weird. I tried the suggestion by @oinopion but it did not work. I imported the project from existing source and it worked.

Weird part is, the setting mentioned by @oinopion was populated with the same value after importing as I had entered manually.

I should note too that I am on Windows and the plugin does NOT like spaces in paths. I created a symlink using mklink to a directory with no spaces and it works a treat.
